### PR TITLE
FS-1666 Add assessments stats endpoint

### PR DESCRIPTION
**Pull request by harryyo**
### Added assessments stats

- Added `/assessments/get-stats/{fund_id}/{round_id}` endpoint
- New flags_by_type query

### CODE REVIEW

**Review by tferns — APPROVED** [review]
Looks good, great work!

**Comment by tferns** [comment]
> Looks good, great work!

Only comment is, I think the logic and code in this PR is sound and good.  But we had a small discussion about this model potentially changing so I'm not sure if that is being done in parallel by @AbuNavsa or @Adam-W1 - if so they may need to also update this endpoint to reflect the model changes.

File: api/routes/assessment_routes.py [hunk 1]
```diff
@@ -114,3 +115,103 @@ def get_sub_criteria_theme_answers(application_id: str, theme_id: str):
...
+            ),
+    # TODO: add resolved if required, after flag_type enum
+    # has been updated to include QUERY_RESOLVED
+    #         "query_resolved": len(
+    #             [
+    #                 1
+    #                 for assessment in assessments
+    #                 if assessment["application_id"] in resolved_assessments
+    #             ]
+    #         ),
```
Comment (harryyo) [inline api/routes/assessment_routes.py:212]: Have just left this commented for now pending update of FlagType enum to contain QUERY_RESOLVED fyi @Adam-W1

Comment (harryyo) [reply api/routes/assessment_routes.py]: Removed

**Pull request merged**